scripts/generate_service.go

```diff
@@ -0,0 +1,305 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+)
+
+type ComputeApiGenVersion int
+
+const (
+	none ComputeApiGenVersion = iota
+	v1
+	v0beta
+)
+
+var OrderedComputeApiVersions = []ComputeApiGenVersion{
```

Comment:
If the constants aren't ordered, we should avoid using iota. If the constants are ordered, I don't understand the need for the slice. :)